File: dotnet-api/Controllers/ApiController.cs
using Microsoft.AspNetCore.Mvc;

namespace dotnet_api.Controllers;

[ApiController]
[Route("/")]
public class ApiController : ControllerBase
{

    [HttpGet("healthcheck")]
    [Produces("application/json")]
    public ActionResult<string> Healthcheck()
    {
        return Ok("Ok");
    }

    [HttpGet("employees")]
    [Produces("application/json")]
    public ActionResult<List<Employee>> GetEmployees()
    {
        var employees = new List<Employee>
        {
            new() { Id = 1, Firstname = "John", Lastname = "Doe" },
            new() { Id = 2, Firstname = "Peter", Lastname = "Parker" },
            new() { Id = 3, Firstname = "Tony", Lastname = "Stark" }
        };
        return Ok(employees);
    }

    [HttpPost("employees")]
    [Produces("application/json")]
    public async Task<ActionResult<int>> CreateEmployee()
    {
        await Task.Delay(300);
        return Ok(42);
    }
}

File: php-laravel/app/Http/Controllers/ApiController.php
<?php

namespace App\Http\Controllers;

use App\Http\Controllers\Controller;
use Illuminate\Http\Request;

class ApiController extends Controller
{
    public function healthcheck()
    {
        return response()->json('Ok');
    }

    public function getEmployees()
    {
        $employees = [
            ['id' => 1, 'firstname' => 'John', 'lastname' => 'Doe'],
            ['id' => 2, 'firstname' => 'Peter', 'lastname' => 'Parker'],
            ['id' => 3, 'firstname' => 'Tony', 'lastname' => 'Stark']
        ];

        return response()->json($employees);
    }

    public function createEmployee(Request $request)
    {
        usleep(300000); // Delay de 300ms

        return response()->json(42);
    }
}
